osu.Game/Online/Chat/DrawableLinkCompiler.cs

```diff
@@ -30,6 +34,12 @@ public partial class DrawableLinkCompiler : OsuHoverContainer
         [Resolved(CanBeNull = true)]
         private OverlayColourProvider overlayColourProvider { get; set; }
 
+        [Resolved]
+        private OnScreenDisplay onScreenDisplay { get; set; }
```

Comment:
This is to fix the test error, but the `#nullable disable` line needs to be removed. See https://github.com/ppy/osu/wiki/Nullable-Reference-Types-(NRT).
```
        private OnScreenDisplay? onScreenDisplay { get; set; }
```